README.md

```diff
@@ -19,9 +19,14 @@ ARCH=amd64 docker build -t kinvolk/calico-hostendpoint-controller:v0.0.2-amd64 -
 ARCH=arm64 docker build -t kinvolk/calico-hostendpoint-controller:v0.0.2-arm64 --build-arg ARCH . && docker push kinvolk/calico-hostendpoint-controller:v0.0.2-arm64
 ```
 
+To build an image for another architecture which is not your native OS architecture there are multiple ways.
+The most convenient one is to install the `qemu-user-static` package on your system, maybe followed by `sudo systemctl restart systemd-binfmt.service` to set up binary translation so that the Linux kernel can run binaries for other architectures.
```

Comment:
```suggestion
The most convenient one is to install the `qemu-user-static` package on your system to set up binary translation so that the Linux kernel can run binaries for other architectures.

>NOTE: You may have to run `sudo systemctl restart systemd-binfmt.service` after installing.
```